hal/halx86/acpi.cmake

```diff
@@ -3,9 +3,16 @@ list(APPEND HAL_ACPI_SOURCE
     acpi/halacpi.c
     acpi/halpnpdd.c
     acpi/busemul.c
+    acpi/madt.c
     legacy/bus/pcibus.c)
 
+if(ARCH STREQUAL "amd64")
+    add_definitions(-DWIN64)
+endif()
+
 add_library(lib_hal_acpi OBJECT ${HAL_ACPI_SOURCE})
+include_directories(include ${REACTOS_SOURCE_DIR}/drivers/bus/acpi/acpica/include)
```

Comment:
Move this at the top of the cmake file.
But then I wonder, whether we need then to still specify the full path to acpi.h below?

Reply:
Yes, we'd still need to specify the full path.